### PR TITLE
bluetooth: tester: Update overlays

### PR DESCRIPTION
**Pull request by mkasenberg**
- The behavior of CONFIG_LOG_DEFAULT_LEVEL was fixed, so lets add to the tester its own config menu for log level.
- Rearrange LE Audio overlays to fix some build warnings about unused CONFIG_. 
-- For nrf52840 apply overlay-le-audio.conf and overlay-le-audio-ctlr.conf
-- For nrf53840 apply just overlay-le-audio.conf.

### CODE REVIEW

**Comment by mkasenberg** [comment]
It seems that Compliance Checks do not like templates. Can I workaround it somehow?

File: tests/bluetooth/tester/Kconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Config options for tester app
+
+# Copyright (c) 2023 Codecoup
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Logger sample application"
```
Comment (Thalley) [inline tests/bluetooth/tester/Kconfig:6]: Not a sample application :)

**Comment by Thalley** [comment]
> It seems that Compliance Checks do not like templates. Can I workaround it somehow?

Hmm, usually that's not a problem. The only difference between your changes and the others in the repo, is that you use `menu` instead of `menuconfig` (or none). 

@theob-pro You are familiar with the logging system; do you have a way to fix this?

**Comment by theob-pro** [comment]
> > It seems that Compliance Checks do not like templates. Can I workaround it somehow?
> 
> Hmm, usually that's not a problem. The only difference between your changes and the others in the repo, is that you use `menu` instead of `menuconfig` (or none).
> 
> @theob-pro You are familiar with the logging system; do you have a way to fix this?

I think this is more a Kconfig problem than a logging system problem.

One thing that I find strange is that you are sourcing `Kconfig.zephyr`. Shouldn't it be the other way around, your Kconfig file being sourced by the Kconfig in the previous folders?

**Comment by theob-pro** [comment]
> > One thing that I find strange is that you are sourcing `Kconfig.zephyr`. Shouldn't it be the other way around, your Kconfig file being sourced by the Kconfig in the previous folders?
> 
> That's how it is done in https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/subsys/logging/logger/Kconfig and https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/subsys/logging/log_api/Kconfig

I had a quick look at the `check_compliance.py` and it look like the example you link encounter the same problem. They added their symbols to the `UNDEF_KCONFIG_WHITELIST`.

https://github.com/zephyrproject-rtos/zephyr/blob/d5f83d2a7f1d3c7031bc2eec6b72e7261234bcba/scripts/ci/check_compliance.py#L645-L646

**Review by sjanc — APPROVED** [review]

**Review by Thalley — APPROVED** [review]

**Pull request merged**